src/js/eventInTimeAware.js

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+/* global fluid, floe */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // A grade that deals in time,
```

Comment:
The "" modelListener is to ensure that any changes to the component model result in an updated `lastModified` time stamp. I agree this is aggressive, and it might make more sense to expect any implementing grade to indicate which events are consequential "modifications" that should result in a `lastModified` change and which are not, similar to what is done for the `pouchPersisted` grade.